pyo3-build-config/src/impl_.rs

```diff
@@ -1554,13 +1561,24 @@ fn default_lib_name_windows(
     abi3: bool,
     mingw: bool,
 ) -> String {
-    if abi3 && !implementation.is_pypy() {
-        WINDOWS_ABI3_LIB_NAME.to_owned()
-    } else if mingw {
-        // https://packages.msys2.org/base/mingw-w64-python
-        format!("python{}.{}", version.major, version.minor)
-    } else {
-        format!("python{}{}", version.major, version.minor)
```

Comment:
Hmm, looks like PyPy uses `pythonXY.lib` to link to `libpypy3.Y-c`/`libpypy3-c`... I need to fix it in `python3-dll-a` first.

Reply:
Fixed in https://github.com/PyO3/python3-dll-a/pull/27